shell/platform/windows/flutter_windows_engine.cc

```diff
@@ -327,18 +327,19 @@ bool FlutterWindowsEngine::Run(std::string_view entrypoint) {
     auto host = static_cast<FlutterWindowsEngine*>(user_data);
     host->OnPreEngineRestart();
   };
-  args.update_semantics_callback = [](const FlutterSemanticsUpdate* update,
```

Comment:
Is the old field not needed/not present anymore?

Reply:
We cannot remove APIs from the embedder API. The old callback still exists but is deprecated. An embedder must provide only one semantic update callback.